docs/content/posts/2023-02-23-release-1.15.4.md

```diff
@@ -0,0 +1,152 @@
+---
+authors:
+- danny: null
+  name: Danny Cranmer
+date: "2023-02-23T17:00:00Z"
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink
+  1.15.
+title: Apache Flink 1.15.4 Release Announcement
+aliases:
+- /news/2023/02/23/release-1.15.4.html
```

Comment:
Since this a new blog post, there's no need to include an alias. Those are meant only for pages that existed before that needed to be redirected to this new URL :)

Reply:
Thanks for the context, will fix